app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickViewModel.kt

```diff
@@ -74,50 +75,67 @@ class QuickViewModel(
         analyticsManager.trackScreen("QuickScreen")
 
         intent {
-            if (isRatesAvailable().not())
+            if (currencyRepo.isRatesAvailable().not()) {
```

Comment:
Please remove unused imports 
```kotlin
import dev.arkbuilders.rate.domain.repo.PortfolioRepo
import dev.arkbuilders.rate.domain.repo.Prefs
```

Reply:
Actually there is no ktlinter in the project. Many files contain unused imports. It would be better if we set up krlinter in separate pr.

Reply:
Okay sounds good

Reply:
Allow me to set up ktlint :)
The UI looks, good work @mdrlzy :+1: